libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -463,12 +460,13 @@ export class PaperHandlerDriver implements PaperHandlerDriverInterface {
   }
 
   async getScannerCapability(): Promise<ScannerCapability> {
-    await this.genericLock.acquire();
-    await this.transferOutGeneric(GetScannerCapabilityCommand, undefined);
-    const transferInResult = await this.transferInGeneric();
-    const { data } = transferInResult;
-    assert(data);
-    return parseScannerCapability(data);
+    return this.genericLock.withLock(async () => {
+      await this.transferOutGeneric(GetScannerCapabilityCommand, undefined);
+      const transferInResult = await this.transferInGeneric();
+      const { data } = transferInResult;
+      assert(data);
+      return parseScannerCapability(data);
+    });
```

Comment:
This is the block that was missing a `.release()` call altogether. Though this method appears to be unused